lcUI/widgets/clicommand.cpp

```diff
@@ -66,6 +70,14 @@ void CliCommand::onReturnPressed() {
         else if (text.indexOf(";") != -1 || text.indexOf(",") != -1) {
             enterCoordinate(text);
         }
+        else if(text.indexOf("=")==4) {
```

Comment:
You should verify with a regular expression here like ([a-zA-Z0-9-_]+)=(.+) and update the correct parameter